MVPSt_2023_Feb_Competition/MVPSt_2023_Feb_Competition/Pages/ManageListingsPage.cs

```diff
@@ -0,0 +1,234 @@
+        //        IWebElement nextpageLink = driver.FindElement(By.TagName("ui buttons semantic-ui-react-button-pagination"));
+
+        //        string nextpageLinkstring = nextpageLink.GetAttribute("");
+        //        if (nextpageLinkstring.Contains("disabled"))
+        //        {
+        //            Console.WriteLine("next page control is disabled");
+        //        }
+        //        else
+        //        {
+        //            Console.WriteLine("next page control is enabled");
+        //        }
+                
+        //        // just giving a default value until solution to looping issue is found along with how to capture row count for every page
+        //        //numberofListings = 1;
+        //        // -- design: 1. count # of rows on current page by adding rows to a total row count, 2. if next page icon is enabled, then click it AND
+        //        // -- perform step 1. again. Must do step 1 a miniumum of 1 time!
+        //        numberofListings = 0;
+        //        //continuecounting = "Yes";
+        //        //do
+        //        //{
+        //            numberofrows = 0;
+        //            IList<IWebElement> tablerow = tableofListings.FindElements(By.TagName("tr"));
+        //            numberofrows = tablerow.Count;
+        //            numberofListings += numberofrows; // adds numberofrows to numberofListings
+        //        //    // if next page icon is enabled, then click it, else set continuecounting to 'No'
+        //        //    if (nextpageLink.Enabled == true)
+        //        //    {
+        //        //        nextpageLink.Click();
+        //        //    }
+        //        //    else
+        //        //    {
+        //        //        continuecounting = "No";
+        //        //    }
+        //        //}
+        //        //// while next page navigation icon is enabled
+        //        //while (continuecounting == "Yes");
+        //        //bool v = nextpageLink.GetAttribute("disabled") = true;
+        //        //while (v)
+        //        //{
+
+        //        //}
+        //    }
+        //    // remove following line after resolving the loop and next page navigation control issue
+        //    //numberofListings = 1;
+        //    return numberofListings;
+
+        //}
+        
+        //public void NavigateToProfileHome()
+        //{
+        //    // find and click the Profile link
+        //    profileLink.Click();
+        //}
+
+        public void NavigateToShareSkill()
+        {
+            // find and click the Share Skill button
+            shareskillButton.Click();
+        }
+
+        // this method is only called by positive test cases containing valid data
+        public bool CompareManageListingsValues(int NumberOfExistingListingsInt, string Category, string Title, string Description, string ServiceType, string SkillTrade, string Active)
+        {
```

Comment:
can you please break this method into smaller methods, this way your code is more readable and easy to understand

Reply:
Removed 1 parameter from parms and created separate methods to check each parameter